values.yaml

```
@@ -280,6 +280,9 @@ webserver:
   # Number of webservers
   replicas: 1
 
+  # Prevent airflow webserver reboot only for Airflow 2.0.0
+  useDefaultAirflowImage: true
```

```suggestion
  # Prevent Airflow webserver reboot only for Airflow 2.0.0
  # AC image with matching Airflow version will be used
  # to run the Webserver. Since Airflow >= 2.0 uses DAG Serialization
  # User image containing DAGs is not required to run Webserver.
  useDefaultAirflowImage: true
```

Can you add this info to https://github.com/astronomer/airflow-chart/blob/master/README.md#parameters too please

---

We should also add docs to https://www.astronomer.io/docs/cloud/stable/deploy/deploy-cli -- not as a scope of this PR but maybe as and additional Github Issue that can be tackled separately.

cc @paolaperaza